app/assets/stylesheets/order.css

```diff
@@ -0,0 +1,70 @@
+.gradient-custom-2 {
+    
+    background-color: #eee;
+    
+    
+    }
+    
+    #progressbar-1 {
+    color: #455A64;
+    }
+    
+    #progressbar-1 li {
+    list-style-type: none;
+    font-size: 13px;
+    width: 33.33%;
+    float: left;
+    position: relative;
+    }
+    
+    #progressbar-1 #step1:before {
+    content: "1";
+    color: #fff;
+    width: 29px;
+    margin-left: 22px;
+    padding-left: 11px;
+    }
+    
+    #progressbar-1 #step2:before {
```

Comment:
progressbar-1 is a bad name for an item. You can also use classes instead of ID to apply styles